### PR TITLE
[ES-1902] Abrupt disconnection of WebSocket is not updating the verification status as Failed

### PR DESCRIPTION
**Pull request by Md-Humair-KK**
Identify if its abnormal disconnection.

If abnormal is confirmed, then update the verified_slot transaction as “FAILED” and save the transaction.

other logic to remove/evict the allotted slot will remain the same.

### CODE REVIEW

File: signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java [hunk 1]
```diff
@@ -69,6 +72,12 @@ public void onConnected(SessionConnectedEvent connectedEvent) {
     public void onDisconnected(SessionDisconnectEvent disconnectEvent) {
         String username = Objects.requireNonNull(disconnectEvent.getUser()).getName();
         log.info("WebSocket Disconnected >>>>>> {}", username);
+        if(disconnectEvent.getCloseStatus()!=null && !disconnectEvent.getCloseStatus().equals(CloseStatus.NORMAL)){
+            IdentityVerificationTransaction transaction =
+                    cacheUtilService.getVerifiedSlotTransaction(username.split(VALUE_SEPARATOR)[1]);
```
Comment (ase-101) [inline signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java:77]: As we are splitting the "value" multiple times, can we split it once and assign it to a variable, and use the variable instead?

File: signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java [hunk 2]
```diff
@@ -69,6 +72,12 @@ public void onConnected(SessionConnectedEvent connectedEvent) {
     public void onDisconnected(SessionDisconnectEvent disconnectEvent) {
         String username = Objects.requireNonNull(disconnectEvent.getUser()).getName();
         log.info("WebSocket Disconnected >>>>>> {}", username);
+        if(disconnectEvent.getCloseStatus()!=null && !disconnectEvent.getCloseStatus().equals(CloseStatus.NORMAL)){
```
Comment (ase-101) [inline signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java:75]: if you reverse the condition you could eliminate null check @Md-Humair-KK . 
if(!CloseStatus.NORMAL.equals(disconnectEvent.getCloseStatus())) try this once.

File: signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java (hunk 1 above)
Comment (ase-101) [reply signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java:77]: And I don't see any Junit test case change or added.

**Review by ase-101 — CHANGES_REQUESTED** [review]
check comments

File: signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java (hunk 2 above)
Comment (Md-Humair-KK) [reply signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java]: Yes @ase-101 Thank you for the information, it worked after inverting the condition without null check.

**Review by ase-101 — APPROVED** [review]

**Pull request merged**